src/Lykke.Service.ExchangeConnector/Exchanges/Concrete/BitMEX/BitMexModelConverter.cs

```diff
@@ -10,11 +11,17 @@
 
 namespace TradingBot.Exchanges.Concrete.BitMEX
 {
-    internal static class BitMexModelConverter
+    internal class BitMexModelConverter: ExchangeConverters
     {
         private const decimal SatoshiRate = 100000000;
 
-        public static PositionModel ExchangePositionToModel(Position position, ICurrencyMappingProvider configuration)
+        public BitMexModelConverter(IReadOnlyCollection<CurrencySymbol> currencySymbols,
+            string exchangeName): base(currencySymbols, exchangeName)
+        {
+
+        }
+
+        public static PositionModel ExchangePositionToModel(Position position)
         {
             return new PositionModel
             {
```

Comment:
Another hardcoded symbol on lines 29 and 49

Reply:
The same reason